src/main/java/n3/arrays/ArrayTask4.java

```diff
@@ -0,0 +1,25 @@
+package n3.arrays;
+
+/*
+- написати програму яка шукає однакові значення масиву ( дублікати )
+*/
+
+public class ArrayTask4 {
+    public static void main(String[] args) {
+        String [] arr = {"qwerty", "txt", "test1", "test2", "nnnnn", "r", "txt", "test3", "ttt", "txt", "txt"};
+        boolean dublicate;
+
+        for (int i = 0; i < arr.length; i++) {
+            dublicate = false;
+            for (int k = 0; k < arr.length; k++){
+                if ( i != k && arr[i] == arr[k]){
```

Comment:
Please, use appropriate function for strings equality comparison.

Reply:
fixed